@here/harp-mapview-decoder/lib/TileDataSource.ts

```diff
@@ -255,8 +260,15 @@ export class TileDataSource<TileType extends Tile> extends DataSource {
 
                 tile.setDecodedTile(tileLoader.decodedTile);
             } else {
-                // empty tiles are traditionally ignored and don't need decode
-                tile.forceHasGeometry(true);
+                if (tileLoader.decodedTile !== undefined) {
+                    // to render at least an empty surface
```

Comment:
Please adhere to the coding style and general language rules:
- fill line length of 100 chars and avoid comments spread over multiple short lines
- no comma at the end of a sentence, but a colon.
- start of a sentence looks better with uppercase...
- these rules have not been applied to all parts of the code, but please follow them in new code.